docs/getting-started/rulesets.md

```diff
@@ -40,6 +40,60 @@ While running it with this object, it will succeed:
 }
 ```
 
+### Formats
+
+Formats are an optional way to specify which API description formats a rule, or ruleset, is applicable to. Currently Spectral supports these two formats:
+
+- `oas2` (this is OpenAPI v2.0 - formerly known as Swagger)
+- `oas3` (this is OpenAPI v3.0)
+
+Specifying the format is optional, so you can completely ignore this if all the rules you are writing apply to any document you lint, or if you have specific rulesets for different formats. If you'd like to use one ruleset for multiple formats, the format key is here to help.
+
+```yaml
+rules:
+  api-servers:
+    description: "OpenAPI `servers` must be present and non-empty array."
+    recommended: true
+    formats: ["oas3"]
+    given: "$"
+    then:
+      field: servers
+      function: schema
+      functionOptions: 
+        schema:
+          items: 
+            type: object
+          minItems: 1
+          type: array
+```
+
+Seeing as the `servers` array only appeared in OpenAPI v3.0, we don't want this rule appearing when Spectral lints OpenAPI v2.0 documents.
+
+Alternatively, formats can be specified at the ruleset level:
+
+```
+```yaml
```

Comment:
Shall we say a word about the fact that the local rule formats definition has a higher priority?
```yaml
formats: ["oas2", "oas3"]
rules:
  api-servers: # this rule would be applied only to oas2
    description: "OpenAPI `servers` must be present and non-empty array."
    recommended: true
    given: "$"
    formats: ["oas2"] 
    then:
      field: servers
      function: schema
      functionOptions: 
        schema:
          items: 
            type: object
          minItems: 1
          type: array
```

Reply:
For now I think I'd like to avoid drowning them in options but if the question comes up we can add it and make a new Formats page which goes more in depth with all the things.

Reply:
Alrighty, sounds good to me.